pulp_smash/tests/rpm/api_v2/test_upload_publish.py

```diff
@@ -22,7 +23,7 @@
     RPM,
     RPM_FEED_URL,
     RPM_URL,
-)
+    DRPM_URL)
```

Comment:
This should be updated to the canonical style of:

```
from flizbang import (
    bar,
    baz,
    biz,
    foo,
)
```

Can happen in follow up commit.